api/src/opentrons/protocol_runner/json_translator.py

```diff
@@ -106,31 +133,41 @@ def _translate_simple_command(
     return translated_obj
 
 
-class JsonCommandTranslator:
-    """Class that translates commands from PD/JSON to ProtocolEngine."""
-
-    def translate(
+class JsonTranslator:
+    """Class that translates commands/liquids from PD/JSON to ProtocolEngine."""
+
+    def translate_liquids(self, protocol: ProtocolSchemaV6) -> List[Liquid]:
+        """Takes json protocol v6 and translates liquids->protocol engine liquids."""
+        liquids = []
+        if protocol.liquids:
+            for liquid_key in protocol.liquids:
+                liquid = protocol.liquids[liquid_key]
+                liquids.append(
+                    Liquid(
+                        id=liquid_key,
+                        displayName=liquid.displayName,
+                        description=liquid.description,
+                        displayColor=liquid.displayColor,
+                    )
+                )
+        return liquids
+
+    def translate_commands(
         self,
         protocol: ProtocolSchemaV6,
     ) -> List[pe_commands.CommandCreate]:
         """Takes json protocol v6 and translates commands->protocol engine commands."""
         commands_list: List[pe_commands.CommandCreate] = []
-        exclude_commands = [
-            "loadLiquid",
-            "moveToSlot",
-        ]
-        commands_to_parse = [
-            command
-            for command in protocol.commands
-            if command.commandType not in exclude_commands
-        ]
+        commands_to_parse = [command for command in protocol.commands]
         for command in commands_to_parse:
```

Comment:
Now that we're not filtering anything out of `protocol.commands`, we can simplify this to:

```suggestion
        for command in protocol.commands:
```